jobs/config.json

```diff
@@ -5016,6 +5016,26 @@
       "sig-autoscaling"
     ]
   },
+  "ci-kubernetes-e2e-gci-gke-autoscaling-gpu": {
```

Comment:
In other tests, we had to specify
```
      "--gke-create-command=beta container clusters create --accelerator=type=nvidia-tesla-k80,count=2",
```
Do you not need to specify that in autoscaling tests?